### PR TITLE
Add a struct to wrap a standard use case

### PR DESCRIPTION
**Pull request by AlexisMontagne**

### CODE REVIEW

**Comment by AlexisMontagne** [comment]
@jlevesy What do you think about it?

File: README.md [hunk 1]
```diff
@@ -83,7 +89,65 @@ func main() {
...
+func main() {
+  c := &utils.StandardCheck{
+    ErrorThreshold: 20.0,
+    WarningThreshold: 10.0,
+    MetricName: "http_call.duration",
+    Value: HTTPCallDuration,
+    CheckMessage: func(v float64) string {
+      return fmt.Sprintf("Duration: %.2fs", v)
+    },
+  }
```
Comment (jlevesy) [inline README.md]: Weird Indent here ?

File: README.md [hunk 2]
```diff
@@ -83,7 +89,65 @@ func main() {
...
+
+	cfg := sensu.NewConfigFromFlagSet(sensu.ExtractFlags())
+
+	t := transport.NewRabbitMQTransport(cfg)
+	client := sensu.NewClient(t, cfg)
+
+	check.Store["http_duration_check"] = &check.ExtensionCheck{c.Check}
+	check.Store["http_duration_metric"] = &check.ExtensionCheck{c.Metric}
```
Comment (jlevesy) [inline README.md]: What about non float64 checks ?

File: README.md [hunk 3]
```diff
@@ -83,7 +89,65 @@ func main() {
...
+
+func HTTPCallDuration() float64 {
+  t0 = time.Now().Unix()
+  resp, err := http.Get("http://example.com/")
+
+  return float64(time.Now().Unix() - t0)
+}
+
+func main() {
+  c := &utils.StandardCheck{
```
Comment (jlevesy) [inline README.md]: :+1: :+1:

File: sensu/client.go [hunk 4]
```diff
@@ -8,7 +8,7 @@ import (
 )
 
 const (
-	CurrentVersion     string        = "0.0.1"
+	CurrentVersion     string        = "0.1.0-dev"
```
Comment (jlevesy) [inline sensu/client.go:11]: You might want to remove the -dev ? as it is a release now ?

File: sensu/utils/standard_check.go [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+func EnvironmentValueOrConst(envVar string, constVal float64) float64 {
+	if v, err := strconv.Atoi(os.Getenv(envVar)); err == nil {
+		return float64(v)
+	}
+
+	return constVal
+}
+
+type StandardCheck struct {
+	// Minimal value to trigger an error
```
Comment (jlevesy) [inline sensu/utils/standard_check.go:20]: OMG, COMMENTS, COMMENTS EVERYWHERE \o/

**Comment by jlevesy** [comment]
Great, really great! LGTM

**Pull request merged**